airflow/config_templates/default_airflow.cfg

```diff
@@ -1236,3 +1236,7 @@ shards = 5
 
 # comma separated sensor classes support in smart_sensor.
 sensors_enabled = NamedHivePartitionSensor
+
+[backfill]
+# function to be called after the backfill is done, path needs to be provided
+on_backfill_callback =
```

Comment:
Thank you for the reply.
Users can provide any kind of function. Path to function definition just needs to be added here.
Users define function on their side, and that function is just called on the end of the backfill (if the path is provided)

Reply:
But you assume single function for all dags...
Say for example in my Airflow cluster I have over 100 Dag authors. How would that work?

Reply:
Regarding comment below (adding branch operator).
In our workflow we have around 190 DAGs, and we're running backfills on daily basis.
Some backfills can run for a couple of days.
So we found it useful to have it "globally", instead of adding it to each DAG.

Reply:
> 

You are right. In this case it can't be overridden.
I'll modify that use-case.